### ISSUE TITLE
Improve `require-g-image-src`

### ISSUE THREAD

**Issue by hacknug** [gridsome/eslint-plugin-gridsome#56]
**Please write rule name what you want to improve:** `require-g-image-src`

**How are the rules improved?**

Vue accepts passing an object directly to [`v-bind`](https://vuejs.org/v2/api/#v-bind). It would be cool to support it besides passing props directly.

**Provide 2-3 code examples that this rule should warn about:**

```vue
// This should be accepted
<g-image v-bind="{ src: '' }" />

// This should be accepted or give a warning
<g-image v-bind="computed" />
<g-image v-bind="method(something)" />

// This should give an error
<g-image v-bind="{}" />
```

**Additional context**
I'm not sure if you could check the returned object from computed properties and methods. If that's possible that should be accepted as mentioned on comment.

**Comment by tyankatsu0105** [comment]
@hacknug 
Nice report!!!
I'll try this👌
